src/main/java/uk/gov/hmcts/reform/civil/model/FastClaimUnavailableDateLRspec.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.civil.model;
+
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.civil.enums.dq.SmallClaimUnavailableDateType;
+import uk.gov.hmcts.reform.civil.validation.groups.UnavailableDateGroup;
+import uk.gov.hmcts.reform.civil.validation.interfaces.IsPresentOrEqualToOrLessThanOneYearInTheFuture;
+
+import java.time.LocalDate;
+
+@Data
+@Builder
+@IsPresentOrEqualToOrLessThanOneYearInTheFuture(groups = UnavailableDateGroup.class)
+
+public class FastClaimUnavailableDateLRspec {
```

Comment:
Looks  similar to the UnavailableDateLRspec. Do we need this

Reply:
remove two files

Reply:
SmallClaimUnavailableDate cant be removed because its being used in Small claim journey, deleted FastClaimUnavailableDateLRspec